extensions-core/datasketches/src/test/java/org/apache/druid/query/aggregation/datasketches/hll/sql/HllSketchSqlAggregatorTest.java

```diff
@@ -207,6 +234,9 @@ public void tearDown() throws Exception
   @Test
   public void testApproxCountDistinctHllSketch() throws Exception
   {
+    // Can't vectorize due to CONCAT expression.
```

Comment:
super nit: but this is actually due to `SUBSTRING` i think

Reply:
btw, I definitely don't think this is worth fixing in this PR, we can just fix whenever we remove this when we add more vectorized string expressions